css/style.css

```diff
@@ -43,54 +42,49 @@ h1 > small {
     font-size: 20px;
     z-index: 1000000000;
 }
-
 .navbar-fixed-top {
     top: 0;
     border-width: 0 0 0 0;
 }
-
-.navbar-default .navbar-nav > li > a {
+.navbar-default .navbar-nav>li>a
+{
     color: #000a91;
 }
-
-.navbar-default .navbar-nav > li > a:hover {
+.navbar-default .navbar-nav>li>a:hover
+{
     color: #000a91;
     background-color: #fff;
 
 }
-
-.navbar-default .navbar-nav > li > a:active {
+.navbar-default .navbar-nav>li>a:active
+{
     color: #af000a;
     background-color: #fff;
 }
-
 #hero {
     position: relative;
 }
-
 .hero-content {
-    height: 100%;
+    height: 0;
     position: relative;
     text-align: center;
     color: #004276;
     z-index: 100000000;
 }
-
 .hero-text, .hero-content > a {
-    transform: translate(0px, 35vh);
+    transform: translate(0px,35vh);
 }
 
 #features > .col-lg-12 .top1 > .col-lg-6 .col-md-6 .col-sm-6 .col-xs-12 {
     position: relative;
 }
-
 #features i.fa {
     position: absolute;
     top: 15px;
     right: 15px;
 }
 
-#cookieInfo {
+#cookieInfo{
```

Comment:
czemu usunięta spacja? ze spacją to jest właśnie poprawne formatowanie cssa. Klasy oraz ID w html piszemy kebab-case, powinno być #cookie-info